assets/scripts/utils/array.js

```diff
@@ -1,101 +1,124 @@
-import { isArray } from './is';
+import { isArray } from './is'
 
-export function addToArray ( array, value ) {
-    const index = array.indexOf( value );
 
-    if ( index === -1 ) {
-        array.push( value );
+/**
+ * Add item of array if doesn't exist
+ * @param {array}       array   - array of items
+ * @param {?string}     value   - value to add to the array
+ * @return {void}
+ */
+
+const addToArray = (array, value) => {
+    if (array.indexOf(value) === -1) {
+        array.push(value)
     }
 }
 
-export function arrayContains ( array, value ) {
-    for ( let i = 0, c = array.length; i < c; i++ ) {
-        if ( array[i] == value ) {
-            return true;
-        }
-    }
 
-    return false;
-}
+/**
+ * Check if array contains a value
+ * @param {array}       array   - array of items
+ * @param {?string}     value   - value to find in the array
+ * @return {boolean}    true if array contains value
+ */
 
-export function arrayContentsMatch ( a, b ) {
-    let i;
+const arrayContains = (array, value) => array.find(item => item === value)
 
-    if ( !isArray( a ) || !isArray( b ) ) {
-        return false;
-    }
 
-    if ( a.length !== b.length ) {
-        return false;
-    }
+/**
+ * Compare two arrays and see if their content match
+ * @param {array}       a   - first array of items
+ * @param {array}       b   - second array of items
+ * @return {boolean}    true if both arrays are the same
+ */
 
-    i = a.length;
-    while ( i-- ) {
-        if ( a[i] !== b[i] ) {
-            return false;
-        }
-    }
+const arrayContentsMatch = (a, b) => a.length === b.length && a.every((v, i) => v === b[i])
 
-    return true;
-}
 
-export function ensureArray ( x ) {
-    if ( typeof x === 'string' ) {
-        return [ x ];
-    }
+/**
+ * Get last item of the given array
+ * @param {array}       array   - array of items
+ * @return {?string}    the last value of the array
+ */
 
-    if ( x === undefined ) {
-        return [];
-    }
+const lastItemOfArray = array => array[array.length - 1]
 
-    return x;
-}
 
-export function lastItem ( array ) {
-    return array[ array.length - 1 ];
-}
+/**
+ * Remove specific value from array
+ * @param {array}       array   - array of items
+ * @param {?string}     value   - value to remove from the array
+ * @return {void}
+ */
 
-export function removeFromArray ( array, member ) {
-    if ( !array ) {
-        return;
+const removeFromArray = (array, value) => {
```

Comment:
Should this be replaced with `Array.prototype.filter()`?

```js
items.filter(item => item !== itemToRemove)
```

Reply:
The filter function doesn't mutate the array, it only return the array filtered

I'm not really sure if someone still uses these helpers (`array.js` / `transform.js` / `is.js`), we can probably delete some unsued functions. ES6 comes in handy for some use case such as this one.